web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcSupportPanelManager/wcSrsPanelTab/wcSrsPanelTab.directive.js

```diff
@@ -167,14 +167,24 @@ angular.module('common').directive('wcSrsPanelTab',
              */
             $scope.$on('intyg.loaded', function(event, content) {
                 reset();
+                if ($scope.config.intygContext.isSigned) {
+                    $scope.srs.isReadOnly = true;
+                    $scope.srs.intygsTyp = content.typ;
+                    $scope.srs.personId = content.grundData.patient.personId;
+                    $scope.srs.vardgivareHsaId = content.grundData.skapadAv.vardenhet.vardgivare.vardgivarid;
+                    $scope.srs.hsaId = content.grundData.skapadAv.vardenhet.enhetsid
+                    if(content.diagnoser.length > 0) {
+                        $scope.srs.originalDiagnosKod = content.diagnoser[0].diagnosKod
+                    }
+                }
                 if (content.grundData.relation.relationKod === 'FRLANG') {
                     $scope.srs.isForlangning = true;
                     // $scope.srs.extensionFromIntygId = content.grundData.relation.relationIntygsId;
                     $scope.srs.userClientContext = 'SRS_FRL';
                     $scope.srs.extensionPredictionFetched = false;
                 } else {
                     $scope.srs.isForlangning = false;
-                    $scope.srs.userClientContext = 'SRS_UTK';
+                    $scope.srs.userClientContext = $scope.config.intygContext.isSigned ? 'SRS_SIGNED' : 'SRS_UTK';
```

Comment:
Should we add SRS_REH for userClientContext in angular aswell?

Reply:
SRS_REH is already added in the readonly part of the code. There are no requirements saying which tab is supposed to be active so I've kept the original placement and logic for active tab.

Reply:
I checked how the current Rehabstod/Webcert-solution looks like. When opening a certificate in RS it doesn't display SRS as default either. I thought the same as @ChrilleSigerhed but the solution you made @mhornfeldt seems to be the correct one.

I guess you can just merge this PR as @ChrilleSigerhed has already approved it. :-)